config.go

```diff
@@ -221,11 +222,65 @@ func genCertPair(certFile, keyFile string) error {
 func newConfigParser(cfg *config, so *serviceOptions, options flags.Options) *flags.Parser {
 	parser := flags.NewParser(cfg, options)
 	if runtime.GOOS == "windows" {
-		parser.AddGroup("Service Options", "Service Options", so)
+		_, _ = parser.AddGroup("Service Options", "Service Options", so)
 	}
 	return parser
 }
 
+// cleanAndExpandPath expands environment variables and leading ~ in the
+// passed path, cleans the result, and returns it.
+func cleanAndExpandPath(path string) string {
```

Comment:
This method seems to be copy/pasted into `cmd/miner/config.go`. Can we remove this duplication?

Reply:
This is intentional, avoids importing function from dcrpool.